### PR TITLE
Fix send method return value

### PR DESCRIPTION
**Pull request by rekmarks**
In the production extension, `ethereum.send(method: string, params?: Array<any>)` returns the entire JSON RPC `response` object, not `response.result`.

We want to keep this to avoid breaking `send`, so this has now been fixed.

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -34,12 +34,12 @@ function createErrorMiddleware () {
   }
 }
 
-// resolve response.result, reject errors
-const getRpcPromiseCallback = (resolve, reject) => (error, response) => {
+// resolve response.result or response, reject errors
+const getRpcPromiseCallback = (resolve, reject, resolveResponse = false) => (error, response) => {
```
Comment (Gudahtt) [inline src/utils.js]: `unwrapResult` might be a better name for this? The promise is getting resolved either way here.

**Review by Gudahtt — DISMISSED** [review]

File: src/utils.js (hunk 1 above)
Comment (Gudahtt) [reply src/utils.js:38]: Well, technically nothing is getting resolved here. The promise is resolved elsewhere, and the response is returned here, unwrapped or not.

Comment (rekmarks) [reply src/utils.js]: Can't argue with that! 83b4220

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**